### DISCUSSION TITLE
How to delete a registered service

### DISCUSSION THREAD

**Discussion by bongmu** [apache/skywalking#9660] [Q&A] [answered]
How to delete a registered service

**Comment by wu-sheng** [comment, marked as answer]
We don't provide that, it would be removed from TTL mechanism automatically.

**Reply by bongmu** [reply]
How long does it take to delete

**Reply by wu-sheng** [reply]
Read docs before submit questions randomly. https://skywalking.apache.org/docs/main/next/en/setup/backend/ttl/
